### ISSUE TITLE
Sequelize cli generated config.json path is not reachable from model/index.js

### ISSUE THREAD

**Issue by techyrajeev** [sequelize/cli#614]
## What you are doing?
I used sequelize-cli to generate models and configs using "init" command.

## Environment

Windows

## File name : index.js inside folder called models as generated by cli
```js
'use strict';

var fs        = require('fs');
var path      = require('path');
var Sequelize = require('sequelize');
var basename  = path.basename(__filename);
var env       = process.env.NODE_ENV || 'development';
var config    = require(__dirname + '/..\config\config.json')[env];
var db        = {};

if (config.use_env_variable) {
  var sequelize = new Sequelize(process.env[config.use_env_variable], config);
} else {
  var sequelize = new Sequelize(config.database, config.username, config.password, config);
}
```

## What do you expect to happen?
When I ran the project using 
npm start . It should fetch db configurations as stored inside config.json file

## What is actually happening?
Error initializing middleware
Error: Cannot find module 'C:\Users\test\Development\test-api\models/..configconfig.json'
    at Function.Module._resolveFilename (module.js:538:15)
    at Function.Module._load (module.js:468:25)

Basically path generated is incorrect

C:\Users\test\Development\test-api\models/..configconfig.json

__Dialect:__ mysql
__Sequelize CLI version: [Node: 8.9.4, CLI: 3.2.0, ORM: 4.32.2]
__Sequelize version:"^4.32.2"

**Comment by tlindener** [comment]
Hi,
you can go into the index.js and change the line which defines config (line8 or something) to this:
var config = require(__dirname + '/../config/config.json')[env]

**Comment by cosmosquark** [comment]
Hello.
I figured to have a go at resolving this issue, see the pull request that I have just opened.

**Comment by sushantdhiman** [comment]
Fixed by #648
